cards/against-medical-advice/card.md

```diff
@@ -67,12 +67,9 @@ When patients leave AMA, providers should do whatever is possible to limit bad m
 
 ## References
 
-- Bitterman RA. Against medical advice: When should you take “no” for an answer? Lecture presented at ACEP Scientific Assembly. Chicago, Oct. 30, 2008.
+1. Bitterman RA. Against medical advice: When should you take “no” for an answer? Lecture presented at ACEP Scientific Assembly. Chicago, Oct. 30, 2008.
 
```

Comment:
We've been using unordered lists for the references. We can change to ordered if that's the preference.